src/main/java/seedu/address/model/Model.java

```diff
@@ -16,9 +15,9 @@
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
-    Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
-    Predicate<Elderly> PREDICATE_SHOW_ALL_ELDERLYS = unused -> true;
-    Predicate<Volunteer> PREDICATE_SHOW_ALL_VOLUNTEERS = unused -> true;
+    Predicate<Elderly> PREDICATE_SHOW_ALL_ELDERLY = unused -> true;
+    Predicate<Volunteer> PREDICATE_SHOW_ALL_VOLUNTEER = unused -> true;
+    Predicate<Pair> PREDICATE_SHOW_ALL_PAIR = unused -> true;
```

Comment:
Tempted to combine all of them into `PREDICATE_SHOW_ALL`..what do you think?

Reply:
This could be a bit technical challenging, because they are used as a filter where specific Predicate<Elderly> / Predicate<Pair> are required...How are you going to combine Pair with the Elderly and Volunteer?